sync_test.go

```diff
@@ -1024,16 +1024,14 @@ func waitForSync(t *testing.T, svc *neutrino.ChainService,
 				"header for %d (%s) from DB", i, hash)
 		}
 		knownBasicHeader, err =
-			correctSyncNode.Node.GetCFilterHeader(&hash,
-				false)
+			correctSyncNode.Node.GetCFilterHeader(&hash, 0)
```

Comment:
Minor nit suggestion: modify this line such that the assignment is on the same line. We can then wrap the two arguments, so something like this:
```go
knownBasicHeader, err = correctSyncNode.Node.GetCFilterHeader(
    &hash, 0,
)
```

This also applies to the same case right below this line.

Reply:
I've reformatted this bit of code.